apps/prairielearn/elements/pl-rich-text-editor/pl-rich-text-editor.py

```diff
@@ -69,6 +70,11 @@ def prepare(element_html, data):
             f'Invalid output format "{output_format}". Must be either "html" or "markdown".'
         )
 
+    counter = pl.get_string_attrib(element, "counter", "none")
+    valid_counters = ["character", "word", "none"]
+    if counter not in valid_counters:
+        raise Exception(f"Invalid {counter=}. Valid options are {valid_counters}")
+
```

Comment:
Consider an enum. See #9955. Bonus points (though not necessary) if you also implement that for the output format.